src/components/index.d.ts

```diff
@@ -755,15 +755,32 @@ declare module 'roo-ui/components' {
     any
   >;
 
+  interface DividerKnownProps
+    extends BaseProps,
+      SS.HeightProps,
+      SS.WidthProps,
+      SS.SpaceProps,
+      SS.ColorProps,
+      SS.DisplayProps {
+    variant?: string;
```

Comment:
```suggestion
    variant?: SS.ResponsiveValue<string>;
```
to allow variant to be responsive like `AlertKnownProps`